.github/workflows/pr.yml

```diff
@@ -584,16 +567,6 @@ jobs:
         working-directory: packages
         run: ./gradlew  :test-base:assembleAndroidTest -Prealm.kotlin.buildRealmCore=false -Prealm.kotlin.mainHost=false
 
-      # We poll the previously started BAAS container to get the hostname of the container to use with Device Farm tests
-      - name: Fetching the BAAS CLI hostname
-        id: baas_cli_poll
-        working-directory: baasaas
-        env:
-          APIKEY: ${{ secrets.BAAS_CLI_API_KEY }}
-        run: |
-          OUTPUT=$(bash cli.sh poll ${{ steps.baas_cli_start.outputs.baas_container_id }} | jq -r '.httpUrl')
-          echo "baas_container_hostname=$OUTPUT" >> $GITHUB_OUTPUT
-      
       - name: Build Android Sync Test Apk
         working-directory: packages
         run: ./gradlew -PsyncTestUrl=${{ steps.baas_cli_poll.outputs.baas_container_hostname }} :test-sync:packageDebug :test-sync:assembleAndroidTest -Prealm.kotlin.buildRealmCore=false -Prealm.kotlin.mainHost=false
```

Comment:
Shouldn't we remove the `syncTestUrl` from here?

Reply:
Good catch 👍